tape_measure_tool.md

```diff
@@ -0,0 +1,90 @@
+# Tape Measure Tool
+
+This document outlines the design for the tape measure tool targeted at
+Ignition Dome and onwards.  The reasoning for the targeting of Dome is largely
+due to the snapping behavior which needs bounding boxes.  If this measurement
+tool is needed for Citadel, I can try to make a prototype version that doesn't
```

Comment:
Let's try not to use the first person on design documents. These documents will live in this repository and be referenced in the future, so we should make them as impersonal and atemporal as possible.

Reply:
Fixed 78f1e9350154489beedb9128a05826fd0e961210